src/stylesheets/components/_card.scss

```diff
@@ -189,6 +189,7 @@ $inner-border-radius: calc(#{$sds-card-border-radius} - #{$sds-card-border-width
   }
 
   &--vertical {
+    text-align: center;
```

Comment:
This does not seem to be related to the action icon. If it not, it may be better to pull this into a separate PR to better track and explain the need for this change.

Reply:
This is for the vertical header. I will move it to a new PR.